test/helpers/user.ts

```diff
@@ -272,39 +288,15 @@ export const createUserAndWallet = async (
     if (!(accountIP instanceof CouldNotFindError) && accountIP instanceof Error)
       throw accountIP
 
-    if (entry.needUsdWallet) {
-      await addWalletIfNonexistent({
-        currency: WalletCurrency.Usd,
-        accountId: account.id,
-        type: WalletType.Checking,
-      })
-    }
+    await addWalletIfNonexistent({
+      currency: WalletCurrency.Usd,
+      accountId: account.id,
+      type: WalletType.Checking,
+    })
   }
 
   if (account instanceof Error) throw account
 
-  if (entry.username) {
```

Comment:
We were but the code has been replaced. We can no longer do this now that we don't have a test_account array for tests (will only be used for prod... until this features is entirely deprecated - because even in prod we don't need that anymore now that we have deeper kratos integration)

You can see the setUsername in the send lightning test